sentry_sdk/types.py

```diff
@@ -0,0 +1,14 @@
+"""
+This module contains type definitions for the Sentry SDK's public API.
+The types are re-exported from the internal module `sentry_sdk._types`.
+
+Disclaimer: Since types are a form of documentation, type definitions
+may change in minor releases. Removing a type would be considered a
+breaking change, and so we will only remove type definitions in major
+releases.
+"""
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from sentry_sdk._types import Event  # noqa: F401
```

Comment:
Lets also re-export the `Hint` type, because those are the two used by before_send/before_send_transaction. 
(I am aware the Hint ist just a dict right now, but we can improve on this later)